lib/logstash/filters/yaml.rb

```diff
@@ -1,4 +1,5 @@
 # encoding: utf-8
+require "logstash-core"
```

Comment:
is this required?

Reply:
Yes, to load some jars like the log4j2. You can see that commenting this doesn't allow the tests to run

Reply:
and this is even weirder since the first line of `yaml_spec.rb` is also `require "logstash-core"`. 
but really this, shouldn't be explicitly required in a plugin.  let's find out why it is here.